apps/src/gamelab/GameLab.js

```diff
@@ -313,6 +313,11 @@ GameLab.prototype.setupReduxSubscribers = function (store) {
     var lastState = state;
     state = store.getState();
 
+    if (state.interfaceMode !== lastState.interfaceMode && state.interfaceMode === GameLabInterfaceMode.ANIMATION
+      && !this.studioApp_.hasContainedLevels) {
```

Comment:
nit: line 317 should be indented two more spaces. also, i suspect line 316 is well over 80 characters long